### PR TITLE
Fix for trailing slash

### PR DESCRIPTION
**Pull request by kbond**
Fix for #7

### CODE REVIEW

File: tests/fixtures/app/Resources/routing.yml [hunk 1]
```diff
@@ -8,3 +8,9 @@ blog_show_comments:
     path: /blog/{slug}/comments
     options:
         i18n: false
+
+homepage:
```
Comment (umpirsky) [inline tests/fixtures/app/Resources/routing.yml]: Put this on top of the file please.

File: tests/fixtures/app/Resources/routing.yml [hunk 2]
```diff
@@ -8,3 +8,9 @@ blog_show_comments:
...
+    path: /
+
+trailing_slash:
```
Comment (umpirsky) [inline tests/fixtures/app/Resources/routing.yml]: Let's put something in the blog spirit here, I suggest:

```yaml
blog_media:
    path: /blog/media/
```

**Review by umpirsky — APPROVED** [review]
Looks great, thanks!

**Comment by kbond** [comment]
Ok, made those changes and rebased to master

**Comment by umpirsky** [comment]
Thanks!

**Pull request merged**